packages/core/content-manager/server/src/services/document-metadata.ts

```diff
@@ -55,25 +88,64 @@ export default ({ strapi }: { strapi: Strapi }) => ({
     }) as unknown as DocumentMetadata['availableStatus'][0] | null;
   },
 
-  // TODO: Modified status
-  async getStatus(uid: Common.UID.ContentType, document: DocumentVersionSelector) {
-    if (document.publishedAt) return 'published';
-    return 'draft';
+  /**
+   * Get the available status of many documents, useful for batch operations
+   * @param uid
+   * @param documents
+   * @returns
+   */
+  async getManyAvailableStatus(uid: Common.UID.ContentType, documents: DocumentVersionSelector[]) {
+    if (!documents.length) return [];
+
+    // The status of all documents should be the same
+    const status = documents[0].publishedAt !== null ? 'published' : 'draft';
```

Comment:
Could we user the CONTENT_MANAGER_STATUS const here?

Reply:
I had mixed feelings here, those are the document service status fields, which do not contain the modified stauts, so I am not sure 🤔